src/main/java/com/venafi/vcert/sdk/connectors/Connector.java

```diff
@@ -11,6 +11,12 @@
 import com.venafi.vcert.sdk.endpoint.ConnectorType;
 import com.venafi.vcert.sdk.policy.domain.PolicySpecification;
 
+
+/**
+ * 
+ * This represents the connector to TPP or Cloud 
```

Comment:
VaaS for Cloud

Reply:
Yes... I've saw it, in fact initially I've put "VaaS" instead of "Cloud", but the rest of the existing javadoc is using the "Cloud" so I've decided to keep it for now, because I have planned to do the release 0.6.2 very soon and in that I will include the update for it.